### DISCUSSION TITLE
Will I Be Able To Use System.Maui in Linux ?

### DISCUSSION THREAD

**Discussion by ZairiAimenDz** [dotnet/maui#178] [General] [answered]
I Have Been Using Linux For Quite Sometime And I'm Really Interested In System.Maui As I Have Tried Xamarin.Forms And Found It Pretty Good.
But There Was One Problem, I Had To Run Windows On VM to Be Able To Use Visual Studio 2019, Which is quite a heavy task considering i use linux because my pc is underperforming.
so the question is, can i develop system.maui apps in Linux ?

Thanks In Advance.

**Comment by mrwcjoughin** [comment, marked as answer]
Yes you will as Maui will be fully supported in vscode and vscode works on
Linux


On Thu, 16 Jul 2020 at 19:44, Aimen Zairi <notifications@github.com> wrote:

> I Have Been Using Linux For Quite Sometime And I'm Really Interested In
> System.Maui As I Have Tried Xamarin.Forms And Found It Pretty Good.
> But There Was One Problem, I Had To Run Windows On VM to Be Able To Use
> Visual Studio 2019, Which is quite a heavy task considering i use linux
> because my pc is underperforming.
> so the question is, can i develop system.maui apps in Linux ?
>
> Thanks In Advance.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dotnet/maui/discussions/178>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACOGERUTQ7KFHNLIJ6IX2KLR3436PANCNFSM4O4U2U3A>
> .
>
-- 
Thanks,

Matthew Joughin
Founder / CTO

+27 81 529 7169
http://www.adaptableapps.net

**Comment by saint4eva** [comment]
Yes.
